skimage/segmentation/quickshift_wrapper.py

```diff
@@ -0,0 +1,55 @@
+from ._quickshift import quickshift as _quickshift
+
+def quickshift(image, ratio=1., kernel_size=5, max_dist=10,
+               return_tree=False, sigma=0, convert2lab=True, random_seed=None):
+    """Segments image using quickshift clustering in Color-(x,y) space.
+
+    Produces an oversegmentation of the image using the quickshift mode-seeking
+    algorithm.
+
+    Parameters
+    ----------
+    image : (width, height, channels) ndarray
+        Input image.
+    ratio : float, optional, between 0 and 1 (default 1).
```

Comment:
remove default here and below, as well. We do not write default values for python functions.